website/ember_osf_web/decorators.py

```diff
@@ -28,3 +28,8 @@ def wrapped(*args, **kwargs):
                 return func(*args, **kwargs)
         return wrapped
     return decorator
+
+
+def storage_i18n_flag_active():
```

Comment:
This doesn't really belong here. But this is just temporary code, so no biggie.

Reply:
Yeah, this file had the necessary stuff imported already and the module was already imported in most files where I needed it so opted here instead of website/utils